### PR TITLE
Power Spectrum Documentation 

### PR DESCRIPTION
**Pull request by Lucia-Fonseca**
## Description
This pull request attempts to update the documentation of the `power_spectrum modules` to Astropy standards. It also attempts to solve overlooked typos and to add plots in the examples section, eventually. It partly addresses issue #168.
## Checklist
- [x] Follow the [Contributor Guidelines](https://github.com/skypyproject/skypy/blob/master/CONTRIBUTING.md)
- [ ] Write unit tests
- [x] Write documentation strings
- [x] Assign someone from your working team to review this pull request
- [x] Assign someone from the infrastructure team to review this pull request
##Additional
This is still a draft.

### CODE REVIEW

File: skypy/power_spectrum/_eisenstein_hu.py [hunk 1]
```diff
@@ -1,3 +1,9 @@
+"""Eisenstein and Hu module.
+
+This module facilitates the computation of the Eisenstein and Hu fitting
+formula for the matter power spectrum.
+"""
+
```
Comment (ntessore) [inline skypy/power_spectrum/_eisenstein_hu.py:6]: This is just a private implementation file and not a submodule.

File: skypy/power_spectrum/_eisenstein_hu.py [hunk 2]
```diff
@@ -10,32 +16,39 @@
...
+    function with baryon acoustic oscillation wiggles. This is described in
+    [1]_ and [2]_.
 
     Parameters
     ----------
-    wavenumber : array_like
-        Array of wavenumbers in units of [Mpc^-1] at which to evaluate the
-        linear matter power spectrum.
+    wavenumber : (nk,) array_like
+        Array of wavenumbers in units of :math:`[Mpc^-1]` at which to evaluate
```
Comment (ntessore) [inline skypy/power_spectrum/_eisenstein_hu.py:27]: This does not render well: image

(I also don't like the angular brackets but that's just me.)

File: skypy/power_spectrum/_eisenstein_hu.py [hunk 3]
```diff
@@ -10,32 +16,39 @@
...
     cosmology : astropy.cosmology.Cosmology
         Cosmology object providing omega_matter, omega_baryon, Hubble parameter
         and CMB temperature at the present day.
     A_s, n_s: float
         Amplitude and spectral index of primordial scalar fluctuations.
     kwmap : float
         WMAP normalization for the amplitude of primordial scalar fluctuations,
-        as described in [3], in units of [Mpc^-1]. Default is 0.02.
+        as described in [3]_, in units of :math:`[Mpc^-1]`.
+        Default is 0.02/Mpc.
```
Comment (ntessore) [inline skypy/power_spectrum/_eisenstein_hu.py:37]: See above. Also the default is `0.02` as it said initially.

File: skypy/power_spectrum/_eisenstein_hu.py [hunk 4]
```diff
@@ -133,14 +147,16 @@ def f(a, b):
...
+    baryon acoustic oscillation wiggles. This is described in
+    [1]_ and [2]_.
 
     Parameters
     ----------
-    wavenumber : array_like
-        Array of wavenumbers in units of [Mpc^-1] at which to evaluate the
-        linear matter power spectrum.
+    wavenumber : (nk,) array_like
+        Array of wavenumbers in units of :math:`[Mpc^-1]` at which to evaluate
```
Comment (ntessore) [inline skypy/power_spectrum/_eisenstein_hu.py:158]: See above.

File: skypy/power_spectrum/_eisenstein_hu.py [hunk 5]
```diff
@@ -199,15 +220,16 @@ def transfer_no_wiggles(wavenumber, A_s, n_s, cosmology):
...
+    matter power spectrum with (or without) baryon acoustic oscillations using
+    formulation from Komatsu et al (2009) in [3]_.
 
     Parameters
     ----------
-    wavenumber : array_like
-        Array of wavenumbers in units of [Mpc^-1] at which to evaluate the
-        linear matter power spectrum.
+    wavenumber : (nk, ) array_like
+        Array of wavenumbers in units of :math:`[Mpc^-1]` at which to evaluate
```
Comment (ntessore) [inline skypy/power_spectrum/_eisenstein_hu.py:231]: See above.

File: skypy/power_spectrum/_growth.py [hunk 6]
```diff
@@ -42,8 +53,9 @@ def growth_function_carroll(redshift, cosmology):
 
     References
     ----------
-    doi : 10.1146/annurev.aa.30.090192.002435
-    """
+    .. [1] doi : 10.1146/annurev.aa.30.090192.002435
```
Comment (ntessore) [inline skypy/power_spectrum/_growth.py:56]: Better to give textual references as everywhere else.

File: skypy/power_spectrum/_growth.py [hunk 7]
```diff
@@ -1,3 +1,9 @@
+"""Growth function module.
+
+This module facilitates the computation of the linear growth function in
+perturbation theory.
+"""
+
```
Comment (ntessore) [inline skypy/power_spectrum/_growth.py:6]: See above.

File: skypy/power_spectrum/_growth.py [hunk 8]
```diff
@@ -56,14 +68,14 @@ def growth_function_carroll(redshift, cosmology):
 
 
 def growth_factor(redshift, cosmology, gamma=6.0/11.0):
-    """Computation of the growth factor.
+    r'''Growth factor.
 
-    Function used to calculate f(z), parametrised growth factor at different
-    redshifts, as described in [1]_ equation 17.
+    Function used to calculate :math:`f(z)``, parametrised growth factor as a
```
Comment (ntessore) [inline skypy/power_spectrum/_growth.py:73]: Double backtick after math.

File: skypy/power_spectrum/_growth.py [hunk 9]
```diff
@@ -98,14 +114,14 @@ def growth_factor(redshift, cosmology, gamma=6.0/11.0):
 
 
 def growth_function(redshift, cosmology, gamma=6.0/11.0):
-    """Computation of the growth function.
+    r'''Growth function.
 
-    Function used to calculate D(z), growth function at different redshifts,
-    as described in [1]_ equation 16.
+    Function used to calculate :math:`D(z)``, growth function at different
```
Comment (ntessore) [inline skypy/power_spectrum/_growth.py:119]: See above.

**Comment by Lucia-Fonseca** [comment]
Done

**Comment by Lucia-Fonseca** [comment]
This is all good now.

**Pull request merged**